src/cpp/dds/topic/DataWriterImpl.cpp

```diff
@@ -580,12 +580,15 @@ void DataWriterImpl::InnerDataWriterListener::on_liveliness_lost(
         fastrtps::rtps::RTPSWriter* /*writer*/,
         const fastrtps::LivelinessLostStatus& status)
 {
+    fastdds::dds::LivelinessLostStatus dds_status;
+    dds_status.total_count = status.total_count;
+    dds_status.total_count_change = status.total_count_change;
```

Comment:
`fastrtps::LivelinessLostStatus` should be another alias for `fasdds::dds::LivelinessLostStatus`, and thus, we can avoid the copy and code duplication.